### DISCUSSION TITLE
Guess access

### DISCUSSION THREAD

**Discussion by saerothh** [CyferShepard/Jellystat#262] [Ideas]
Hello, I’m not sure if it would be very difficult to implement a guest mode for any Jellyfin user. This means they could view statistics and data but wouldn’t be able to modify settings or see the names of other users, which would appear as nameless or anonymous. Thank you, the app is amazing!

**Comment by TGRubilex** [comment]
Any idea if this is going to be added eventually? My users are begging me for a stats page but I don't know how I can share them this when it has full settings access.
